src/vsr/replica.zig

```diff
@@ -1393,14 +1393,11 @@ pub fn ReplicaType(
             assert(message.header.op > self.op);
             assert(message.header.op > self.commit_min);
```

Comment:
Maybe add a `assert(message.header.op > message.header.commit);` to emphasize that the `self.advance_commit_max(message.header.commit, @src().fn_name);` below is ok even though we won't have loaded `message.header` into our journal yet.

Reply:
advancing commit_max should be ok even if we don't have a header? We allow `maybe(self.op < self.commit_max)`

Reply:
Ah, good point!